.gitbook.yaml

```diff
@@ -5,6 +5,7 @@ structure:
   summary: SUMMARY.md
   
 redirects:
+  $: basics/what-is-filecoin/README.md
```

Comment:
@barbaraperic I can't find in the gitbook docs what the `$` means for redirects. Do you have a link?

Reply:
Ah well it turns out that `https://docs.filecoin.io/$` already redirects to `https://docs.filecoin.io/basics/what-is-filecoin`, I guess gitbook handles it automatically. Not sure why it appears in the broken url list...